bigframes/core/__init__.py

```diff
@@ -94,6 +95,7 @@ def __init__(
         hidden_ordering_columns: Optional[Sequence[ibis_types.Value]] = None,
         ordering: ExpressionOrdering = ExpressionOrdering(),
         predicates: Optional[Collection[ibis_types.BooleanValue]] = None,
+        api_methods: List[str] = [],
```

Comment:
Don't use mutable objects for default values. https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

In fact, ArrayValue is intended to be immutable. A tuple would be more appropriate. Sequence is read-only. We could use that for the type annotation if we want to also accept list arguments. https://docs.python.org/3/library/typing.html#typing.Sequence

Reply:
Done using Sequence.